src/main/scala/com/microsoft/hyperspace/index/sources/default/DefaultFileBasedRelationMetadata.scala

```diff
@@ -36,4 +36,6 @@ class DefaultFileBasedRelationMetadata(metadata: Relation) extends FileBasedRela
   override def enrichIndexProperties(properties: Map[String, String]): Map[String, String] = {
     properties
   }
+
+  override def supportsUserSpecifiedSchema: Boolean = true
```

Comment:
nit: for consistency
```suggestion
  override def supportUserSpecifiedSchema: Boolean = true
```